# Forbid the use number `666`

The use of the number `666` is a bad omen.

The number of the beast (Greek: Ἀριθμὸς τοῦ θηρίου, Arithmos tou Thēriou), also known as The devil's number is a term in the Book of Revelation, of the New Testament, that is associated with the Beast of Revelation in chapter 13, verse 18. In most manuscripts of the New Testament and in English translations of the Bible, the number of the beast is six hundred and sixty-six or χξϛ (the number not being representable by the repetition of 6 or ϛ three times – Greek for 6 is ϛ, but ϛϛϛ is not six hundred sixty-six).

Correct it for God's sake.


### Fail

```js
const a = 'Table number is 16662';

```

```js
const myVariable6661 = (a) => a*2;

```


### Pass

```js
const a = 7;
```
